.github/workflows/ansible-lint.yml

```diff
@@ -0,0 +1,31 @@
+---
+name: ansible-lint
+on:
+  push:
+    tags:
+      - v*
+    branches:
+      - main
+  pull_request:
+
+jobs:
+  ansible-lint:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out the codebase.
+        uses: actions/checkout@v4
+
+      - name: Set up Python 3.
+        uses: actions/setup-python@v5
+        with:
+          python-version: "3.x"
+
+      - name: Run ansible-lint
+        uses: ansible/ansible-lint@v25.7.0
```

Comment:
Personally I think, actions maintained by Ansible should be safe. I can imagine that some scanners might claim that we should replace this version number with a commit hash.

Reply:
At the moment no scanner complained about it. So I would leave it like this.

Reply:
Fine with me.